package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Foreman isn't really a node module, these are just dependencies needed to build the webpack bundle. 'dependencies' are the asset libraries in use and 'devDependencies' are used for the build process.",
   "private": true,
   "engines": {
-    "node": ">14.0.0 <16.0.0"
+    "node": ">=18.0.0 <21.0.0"
```

Comment:
I din't know about Debian building. If it works, does it make sense to keep it >= 14?

@evgeni I recall Debian building was less flexible in NodeJS selection. Or did we fix that in pbuilder?

Reply:
It's actually more flexible thanks to us using nodesource ;)

And yes we fixed that in https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/slave/files/pbuilder_F66-add-nodesource-repos